src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -266,13 +272,14 @@ val qualityFormat: Option[FastqQualityFormat] = None,
  @arg(doc="Maximum allowable number of no-calls in a barcode read before it is considered unmatchable.") val maxNoCalls: Int = 2,
  @arg(doc="The sort order for the output sam/bam file (typically unsorted or queryname).") val sortOrder: SortOrder = SortOrder.unsorted,
  @arg(doc="The SAM tag for any molecular barcode.  If multiple molecular barcodes are specified, they will be concatenated and stored here.") val umiTag: String = ConsensusTags.UmiBases,
- @arg(doc="The platform unit (typically `<flowcell-barcode>-<samle-barcode>.<lane>`)") val platformUnit: Option[String] = None,
+ @arg(doc="The platform unit (typically `<flowcell-barcode>-<sample-barcode>.<lane>`)") val platformUnit: Option[String] = None,
  @arg(doc="The sequencing center from which the data originated") val sequencingCenter: Option[String] = None,
  @arg(doc="Predicted median insert size, to insert into the read group header") val predictedInsertSize: Option[Integer] = None,
  @arg(doc="Platform model to insert into the group header (ex. miseq, hiseq2500, hiseqX)") val platformModel: Option[String] = None,
  @arg(doc="Comment(s) to include in the merged output file's header.", minElements = 0) val comments: List[String] = Nil,
  @arg(doc="Date the run was produced, to insert into the read group header") val runDate: Option[Iso8601Date] = None,
- @arg(doc="Output gzipped FASTQs (`.fastq.gz`) instead of BAM files") val outputFastqs: Boolean = false
+ @arg(doc="Output gzipped FASTQs (`.fastq.gz`) instead of BAM files") val outputFastqs: Boolean = false,
+ @arg(doc="Output FASTQs according to Illumina standards, for example, for upload to the BaseSpace Sequence Hub") val illuminaStandards: Boolean = false
```

Comment:
Could `name` or `naming` get added in there somewhere?  Just to make it clear this only effects file names, and not e.g. quality scaling?